mixer/cmd/config.go

```diff
@@ -96,11 +98,49 @@ var configSetCmd = &cobra.Command{
 	},
 }
 
+var configGetCmd = &cobra.Command{
+	Use:   "get <property>",
+	Short: "Get a property in the config file",
```

Comment:
The phrasing of the Short message makes it sound like there is only one config file, but there are two. Also, I think it would be useful to explicitly write which files are config files in the help message. It may be confusing to users that mixer.state is a config file because it does not use the .conf suffix.

Reply:
Indeed, that is the short version from the old PR. I'll update.